spec/mercure.md

```diff
@@ -240,11 +112,11 @@ partial update by using formats such as JSON Patch [@RFC6902] or JSON Merge Patc
 All other properties defined in the Server-Sent Events specification **MAY** be used and **MUST** be
 supported by hubs.
 
-The resource **SHOULD** be represented in a format with hypermedia capabilities such as
+The resource **MAY** be represented in a format with hypermedia capabilities such as
```

Comment:
Why the change?

Reply:
I want to relax the hypermedia part. With the introduction of the regexp matcher, hypermedia just becomes a use case among others.